.idea/gradle.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Ale-Gor can you please remove `.idea` files from the pull request? They are your personal IDE config